pkgs/applications/editors/jetbrains/source/build.nix

```
@@ -187,28 +195,35 @@ let
       repoUrl = "https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies";
       groupId = builtins.replaceStrings [ "." ] [ "/" ] "org.jetbrains.kotlin";
       artefactId = "kotlin-jps-plugin-classpath";
-      version = "1.9.22";
+      version = "2.0.21-RC";
     in
     fetchurl {
       url = repoUrl + "/" + groupId + "/" + artefactId + "/" + version + "/" + artefactId + "-" + version + ".jar";
-      hash = "sha256-ZPfEceGoIChDmjIAjjhDZpyMWQ7/DtP9Ll4YIrZN+PM=";
+      hash = "sha256-jFjxP1LGjrvc1x2XqF5gg/SeKdSFNefxABBlrYl81zA=";
     };
 
     targetClass = if buildType == "pycharm" then "intellij.pycharm.community.build" else "intellij.idea.community.build";
     targetName = if buildType == "pycharm" then "PyCharmCommunityInstallersBuildTarget" else "OpenSourceCommunityInstallersBuildTarget";
 
+  xplat-launcher = fetchzip {
+    url = "https://cache-redirector.jetbrains.com/intellij-dependencies/org/jetbrains/intellij/deps/launcher/242.22926/launcher-242.22926.tar.gz";
+    hash = "sha256-ttrQZUbBvvyH1BSVt1yWOoD82WwRi/hkoRfrsdCjwTA=";
+    stripRoot = false;
+  };
+
 in
 stdenvNoCC.mkDerivation rec {
   pname = "${buildType}-community";
   version = buildVer;
   name = "${pname}-${version}.tar.gz";
   inherit src;
-  nativeBuildInputs = [ p7zip jbr jps-bootstrap ];
+  nativeBuildInputs = [ p7zip jbr jps-bootstrap breakpointHook ];
```

```suggestion
  nativeBuildInputs = [ p7zip jbr jps-bootstrap ];
```
that's just for debugging